### DISCUSSION TITLE
How to choose the training set for large systems

### DISCUSSION THREAD

**Discussion by Zhang-pchao** [deepmodeling/deepmd-kit#582] [Q&A] [answered]
For studying reactions involving a large number of small molecules, clusters can be selected as the training set.

But for periodic systems such as metal surfaces and MOFs, it is not possible to simply select clusters. Does it mean that the entire system is used as a training set ? (it will be time-consuming for the entire system to calculate thousands of energy and force with DFT) Is there any suitable way to deal with it?

**Comment by tuoping** [comment, marked as answer]
I don't quite understand your question. Why don't you use smaller cells with periodic condition as training set? If you aim to study amorphous structures, virious cell strucuture type like fcc, bcc, etc. with perturbation is normally enough.

**Comment by tuoping** [comment]
Here is an excellent work on liquid. 
https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.121.265701

**Reply by Zhang-pchao** [reply]
Thank you for your reply! I understand how to build a training set for a periodic system.

If I want to study the large cluster structure of a periodic system (taken out a part of the conventional cell and saturate the dangling bonds with hydrogen atoms), in addition to adding the various cell structures to the training set, should I also consider the small cluster structure with hydrogen atoms?

**Comment by tuoping** [comment]
Not necessary in principle. For systems with surface, I recommend using dpgen with plumed. You can refer to this work: M Yang, L Bonati, D Polino, M Parrinello, Using metadynamics to build neural network potentials for reactive events: the case of urea decomposition in water, Ctalysis Today,(2021)
